insights/parsers/netstat.py

```diff
@@ -744,4 +745,8 @@ class SsTUPNA(SsTULPN):
         >>> ssa.data[2]['State']
```

Comment:
Your examples don't correspond to the example data you show, row 2 state is `UCONN`.

Reply:
Removed indexing based tests, as it can be different on different python version